File: app/src/main/java/com/upclicks/ffc/models/requests/UpdateProfileRequest.java
package com.upclicks.ffc.models.requests;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UpdateProfileRequest {
    @SerializedName("Gender")
    @Expose
    private String gender;
    @SerializedName("birthDate")
    @Expose
    private String birthDate;
    @SerializedName("Address")
    @Expose
    private String address;
    @SerializedName("Name")
    @Expose
    private String name;
    @SerializedName("Surname")
    @Expose
    private String surname;
    @SerializedName("PhoneNumber")
    @Expose
    private String phoneNumber;
    @SerializedName("CityId")
    @Expose
    private String cityId;
    @SerializedName("emailAddress")
    @Expose
    private String emailAddress;

    public void setGender(String gender) {
        this.gender = gender;
    }

    public void setBirthDate(String birthDate) {
        this.birthDate = birthDate;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setSurname(String surname) {
        this.surname = surname;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public void setCityId(String cityId) {
        this.cityId = cityId;
    }

    public void setEmailAddress(String emailAddress) {
        this.emailAddress = emailAddress;
    }
}

File: app/src/main/java/com/upclicks/ffc/ui/authentication/model/request/UpdateProfileRequest.kt
package com.upclicks.ffc.ui.authentication.model.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UpdateProfileRequest {
    @SerializedName("emailAddress")
    @Expose
    var emailAddress: String? = null

    @SerializedName("phoneNumber")
    @Expose
    var phoneNumber: String? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("surname")
    @Expose
    var surname: String? = null
    @SerializedName("address")
    @Expose
    var address: String? = null
    @SerializedName("birthdate")
    @Expose
    var birthdate: String? = null
    @SerializedName("cityId")
    @Expose
    var cityId: String? = null
    @SerializedName("governorateId")
    @Expose
    var governorateId: String? = null
    @SerializedName("gender")
    @Expose
    var gender: Int? = null
}